pkg/client/client.go

```diff
@@ -486,6 +486,14 @@ type CreateContainerConfig struct {
 
 	// CleanupCmd is the command that will be executed once the container exits
 	CleanupCmd []string
+
+	// RuntimeArgs are the additional arguments passed to the create runtime call
+	// before the command. e.g: crun --runtime-arg create
+	RuntimeArgs []string
+
+	// RuntimeOpts are the additional arguments passed to the create runtime call
+	// after the command. e.g: crun create --runtime-opt
+	RuntimeOpts []string
```

Comment:
I'm not sure about the naming args vs opts. How about having something like:
```go
type Arguments struct {
	// Global options to be added to the runtime call.
	Global  []string

	// Command options added to the runtime call.
	Command []string
}
```

Beside that, runc states that `global options` are our `RuntimeArgs` while `arguments` are our defined `RuntimeOpts`:
```
USAGE:
   runc [global options] command [command options] [arguments...]
```

Reply:
good idea! I agree it was hard to interpret. I have changed to `GlobalArgs` and `CommandArgs`